client/components/app.jsx

```diff
@@ -1,8 +1,37 @@
+/* eslint-disable no-console */
```

Comment:
Please remove this and your extra console logs. Console log is only for debugging, we never turn it in professionally.